scripts/cluster-install/install_load_balancer.sh

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env bash
+# Abort on any failures
+set -e
```

Comment:
btw, personally i like `set -eux` as boilerplate in bash scripts, i think it's more useful than just `-e` (`-u` also makes it error out when using undefined variables, and `-x` shows each command when it runs it, which is useful for debugging)

Reply:
Done, thanks.